DMCompiler/DM/DMProc.cs

```diff
@@ -571,6 +571,12 @@ public void Pop()
             WriteOpcode(DreamProcOpcode.DeleteObject);
         }
 
+        public void CreateMultidimensionalList(int count) {
+            GrowStack(1);
```

Comment:
This pops `count` values off the stack and pushes 1
```suggestion
            ShrinkStack(count - 1);
```